sourcecode/apis/contentauthor/app/Libraries/DataObjects/EditorConfigObject.php

```diff
@@ -19,6 +19,7 @@ class EditorConfigObject
 
     public $locked = false;
     public $pulseUrl = null;
+    public $editorLanguage;
```

Comment:
Would like to see this typed + the expected language format (I believe this would be the same format used by the HTML `lang` attribute?). CreateTrait might cause problems with real types, so it's fine if this is just a docblock.